charts/barman/values.yaml

```diff
@@ -30,16 +30,46 @@ persistence:
     storageClass: ""
 
 barman:
+  # barman.backupMethod -- Barman backup method
+  backupMethod: postgres
+  # barman.backupOptions -- Barman backup options
+  backupOptions: concurrent_backup
+  # barman.barmanHome -- Barman home directory
+  barmanHome: /var/lib/barman
```

Comment:
`/home/barman` vs. `/var/lib/barman`?

Reply:
renamed to `backupDir` to set `barman home` in barman's config, $HOME is set to `/home/barman` within the container which is not configurable

Reply:
Thanks